src/database/DataTypes/RequisitionItem.js

```diff
@@ -108,15 +107,11 @@ export class RequisitionItem extends Realm.Object {
    * @return  {number}
    */
   get ourStockOnHand() {
-    // TODO: this.linkedTransactionItem and this.item can both be null or undefined, unhandled
-    // possible TypeError.
+    if (!this.linkedTransactionItem || !this.item) return null;
```

Comment:
Is this correct? We want to return null if either `linkedTransactionItem` OR `item` are null? Shouldn't this be return null only if BOTH are null?

Reply:
Using AND would not return if item is defined but linkedTransactionItem is not. Then the next line would TypeError crash the app?

Reply:
Yeah, the next lines would have to change. What I'm asking is, it looks like we returns values based on which fields are defined, but we return prematurely if neither one of their parent objects are defined. Shouldn't we check the parent items in the same way, rather than returning prematurely if neither are defined?

Reply:
Ah I see, will address